app/controllers/api/auth/episodes_controller.rb

```diff
@@ -28,6 +28,6 @@ def sorted(res)
   end
 
   def resources_base
-    @resources_base ||= super.merge(authorization.token_auth_episodes)
+    authorization.token_auth_episodes
```

Comment:
I ran into a bug, where memoizing this would break things in the auth endpoints once I enabled the "admin token".  Inside `hal_api-rails` it was checking for pagination - but since we memoized `resources_base` without pagination - it would error.

In any case - not memoizing doesn't appear to hurt anything?  I don't see any additional SQL being run.

Reply:
Okay, I may have slightly mis-analyzed this... the error thrown was:

```
NoMethodError (undefined method `prev_page' for #<ActiveRecord::Relation ...
```

So that's the [CollectionPaging](https://github.com/PRX/hal_api-rails/blob/main/lib/hal_api/representer/collection_paging.rb#L15) representer trying to render a link.  AND importantly - this was only happening on the `auth/podcasts#index`, not on this `auth/episodes#index`.  So, it's related to how podcasts was defining `@podcasts ||=` whereas this defines `@resources_base ||=`.

I _think_ what was happening was, [this logic](https://github.com/PRX/hal_api-rails/blob/main/lib/hal_api/controller/resources.rb#L62) was grabbing the memoized `@podcasts` variable, which didn't have paging on it.  So I _should_ be able to fix this by just changing the podcasts controller to `@resources_base ||=`.

That make sense?

Reply:
Update: memoization happens in other places.  This seems to be related to a combination of memoizing that model name variable, having index caching enabled, and the index cache path manually constructing the filtered/paged/sorted query.

In any case, the best path seems to be _not_ memoizing here.